Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin -
 WORKDIR /app
 COPY . .
 
+RUN composer update
```

Comment:
This `composer update` should not be here. The following line `composer install` installs packages from the version controlled lock file. (The `update` operation will install different package versions than expected.)